### PR TITLE
[EAGLE-3652] - vLLM example

### PR DESCRIPTION
**Pull request by phatvo9**

### CODE REVIEW

File: clarifai/models/model_serving/examples/vllm/example/1/test.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  3. If your input is `image` and you have set custom size of it when building model repository,
+  call `self.preprocess(image)` to obtain correct resized input
+  4. Run this test by calling
+  ```bash
+  pytest ./your_triton_folder/1/test.py
+  #to see std output
+  pytest --log-cli-level=INFO  -s ./your_triton_folder/1/test.py
+  ```
+
+  ### Examples:
```
Comment (ackizilkale) [inline clarifai/models/model_serving/examples/vllm/example/1/test.py:24]: We need to update these examples if we will show them here. text_to_image and visual_classifier are unlikely helpful.

**Review by ackizilkale — CHANGES_REQUESTED** [review]
Can we update the examples on the test page?

File: clarifai/models/model_serving/examples/vllm/example/1/test.py (hunk 1 above)
Comment (phatvo9) [reply clarifai/models/model_serving/examples/vllm/example/1/test.py:24]: The examples in the test.py was generated automatically when initializing model repository.

**Review by ackizilkale — APPROVED** [review]
Looks good!

**Pull request merged**